oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml

```diff
@@ -0,0 +1,147 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# menus: the menus of the left navigation bar
+#   - name: the name of the menu item
+#     icon: icon name
+#     path: link path
+#     dashboard: related dashboard name, the menu item should be activated when the dashboard value is empty or dashboard related scope has services
+#     menus: sub menus of the current item
+
+menus:
+  - name: General Service
+    icon: chart
```

Comment:
Where are the definitions of these icons? Should UI implement this part?

Reply:
Yes, these value is the current dashboard icon name.

Reply:
They seem in low readability. Please rename them. @Fine0830 Please reflect the UI icon names to adopt this PR change.

Reply:
So, I need to wait for @Fine0830 updates, then I update the names? If It's not defined there, the UI doesn't know what kind of icon should be showing.

Reply:
You should update first. UI follows only.